ui/src/components/integrationFormFields/integrationFormFields.tsx

```diff
@@ -23,6 +23,17 @@ export const IntegrationFormFields: FC<IntegrationFormFieldsInterface> = (props)
     });
   }, []);
 
+  const projectIdValidator = (value: string) => {
+    const trimmedValue = value ? value.trim() : value;
```

Comment:
We can provide a format function to the FIeldElement, and trim the value in that function.